chall03/vkuikka.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+	"strconv"
+	"io/ioutil"
+)
+
+func	getData(addr string) string {
+	fmt.Println(addr)
+	response, err := http.Get(addr)
+	if err != nil {
+		return ("error")
+	}
+	print(string(response.StatusCode) + response.Status + "\n")
+	responseData, err := ioutil.ReadAll(response.Body)
+	if err != nil {
+		return ("error")
+	}
+	responseString := string(responseData)
+	fmt.Println("answer: " + responseString)
+	return responseString
+}
+
+func	sendResponse(response string, addr string) bool {
+	values_str := strings.Split(response, " ")[0]
+	values := strings.Split(values_str, ",")
+	r, _ := strconv.Atoi(strings.Split(values[1], "=")[1])
+	g, _ := strconv.Atoi(strings.Split(values[2], "=")[1])
+	b, _ := strconv.Atoi(strings.Split(values[3], "=")[1])
```

Comment:
You should avoid as much as possible to ignore error return in Go, and always check it. 

It can be repetitive, but it's better to check that the convert worked for example here.